AEPServices/Sources/dataqueue/SQLiteDataQueue.swift

```diff
@@ -116,7 +109,7 @@ class SQLiteDataQueue: DataQueue {
                 disconnect(database: connection)
             }
             let deleteRowStatement = """
-            DELETE FROM \(SQLiteDataQueue.TABLE_NAME) order by id limit 1;
+            DELETE FROM \(SQLiteDataQueue.TABLE_NAME) order by id limit \(n);
```

Comment:
nit: https://github.com/adobe/aepsdk-core-ios/pull/453/files#diff-4d8f660abc14b917eaee3faf76596aada2a9b34f04f6fa34a9b49be88ccc978cR79, the sql here are all uppercased, so it'd be better to keep them consistent.